html/captive-portal/lib/captiveportal/PacketFence/Controller/Activate/Email.pm

```diff
@@ -87,6 +87,7 @@ sub code : Path : Args(1) {
         node_modify($c->portalSession->clientMac, unregdate => $unregdate);
         pf::activation::set_status_verified($code);
         $c->stash(
+            title => "Access granted",
```

Comment:
i18n or directly in the template IMO

The configuration will get heavy for something that is never changed